src/docs/GetStarted/Developers/Installation/index.mdx

```diff
@@ -3,35 +3,35 @@ title: 'Installation'
 order: 1
 ---
 
-### Requirements :
+### Requirements:
 
 Installing Mozaic require `node v10 +`
 
-As for now the `@mozaic-ds/**` repos are public registries on NPM.
-You need to be granted and access to the repos.
+As for now, the `@mozaic-ds/**` repositories are public registries on NPM.
```

Comment:
sorry I forgot, the sentence should be (no _the_ with plurals in English 😉):
As for now, `mozaic-ds/**` repositories are public registries on NPM.